wsf_scraping/spiders/base_spider.py

```diff
@@ -29,3 +32,16 @@ def _check_headers(self, response_headers,
                        desired_extension=b'application/pdf'):
         content_type = response_headers.get('content-type', '').split(b';')[0]
         return desired_extension == content_type
+
+    def _save_file(self, url, response_body):
+        filename = os.path.basename(urlparse(url).path).lower()
```

Comment:
Why do you care about lowercasing? That can have surprising effects…

Reply:
True. I need it to check if the filename ends with `.pdf`, but I should only check the lowercased filename, not change it to lowercase.